cmd/evm/internal/t8ntool/flags.go

```diff
@@ -30,10 +30,13 @@ var (
 		Name:  "trace",
 		Usage: "Output full trace logs to files <txhash>.jsonl",
 	}
-	TraceDisableMemoryFlag = &cli.BoolFlag{
-		Name:  "trace.nomemory",
-		Value: true,
-		Usage: "Disable full memory dump in traces (deprecated)",
+	TraceTracerFlag = &cli.StringFlag{
+		Name:  "trace.tracer",
```

Comment:
Would be nice to have something better than `trace.tracer` and `trace.tracer.config` for these fields...

Reply:
Or we can just use `tracer` and `tracerConfig`, the same as `debug_traceXXX` API

Reply:
I think `trace.tracer` is fine. `trace` means this flag belongs to tracer group. `tracer` specifies the name of used tracer.